KicadModTree/nodes/base/Pad.py

```diff
@@ -50,13 +149,20 @@ class Pad(Node):
           drill-size of the pad
         * *radius_ratio* (``float``) --
           The radius ratio of the rounded rectangle.
-          Ignored for every other shape.
+          Ignored for every shape except round rect.
         * *maximum_radius* (``float``) --
           The maximum radius for the rounded rectangle.
           If the radius produced by the radius_ratio parameter for the pad would
-          exceed the maximum radius, the ratio is reduced to limit the ratio.
+          exceed the maximum radius, the ratio is reduced to limit the radius.
           (This is useful for IPC-7351C compliance as it suggests 25% ratio with limit 0.25mm)
-          Ignored for every other shape.
+          Ignored for every shape except round rect.
+        * *round_radius_exact* (``float``) --
+          Set an exact round radius for a pad.
+          Ignored for every shape except round rect
+        * *round_radius_handler* (``RoundRadiusHandler``) --
+          An instance of the RoundRadiusHandler class
+          If this is given then all other round radius specifiers are Ignored
```

Comment:
Is the capital "I" intentional? This is repeated in other files too.

Reply:
fixed

Reply:
Only fixed in this file. As I mentioned, it's repeated elsewhere. I see three other capital "I".